plugins/filter_ecs/ecs.c

```diff
@@ -1393,7 +1393,7 @@ static void mark_tag_failed(struct flb_filter_ecs *ctx,
 
     ret = flb_hash_get(ctx->failed_metadata_request_tags,
                        tag, tag_len,
-                       (void *) val, &val_size);
+                       (void **) &val, &val_size);
```

Comment:
code below is still buggy, need to alloc memory for hash table to copy over in all cases